docs/spec_draft.md

```diff
@@ -343,6 +344,78 @@ logical operation.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.batch_norm_grad
+
+### Semantics
+
+Computes gradients of `Batch Normalizing Transform` for `operand`, `scale` and
```

Comment:
operand `offset ` does not exists ?

Reply:
grad_offset is computed but offset is not required to compute it or any of `grad_operand` or `grad_scale` for that matter.